ape_solidity/compiler.py

```diff
@@ -270,3 +270,36 @@ def load_dict(data: Union[str, dict]) -> Dict:
                 contract_types.append(ContractType.parse_obj(contract_type))
 
         return contract_types
+
+    def get_imports(
+        self, contract_filepaths: List[Path], base_path: Optional[Path]
+    ) -> Dict[str, List[str]]:
+        def import_str_to_paths(path: str, contract_filenames) -> List[str]:
+            path = path.replace("import", "").replace('"', "").replace(";", "").strip()
+            filename = path.split("/")[-1]
```

Comment:
If you make the `pathlib.Path` first, you can just do `path.name` as the filename.
This helps with operating system differences of `/`